src/runtime_src/core/common/drv/xrt_cu.c

```diff
@@ -634,7 +642,11 @@ int xrt_cu_intr_thread(void *data)
 		process_pq(xcu);
 	}
 	xrt_cu_disable_intr(xcu, CU_INTR_DONE | CU_INTR_READY);
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 16, 0)
```

Comment:
The removal of `del_timer_sync()` already happened in v6.15 (multiple occurrences in multiple files).

https://github.com/torvalds/linux/commit/8fa7292fee5c5240402371ea89ab285ec856c916